rql/query/src/main/java/org/eclipse/ditto/rql/query/criteria/CriteriaFactory.java

```diff
@@ -152,6 +152,14 @@ default Criteria nor(final Criteria criteria) {
      */
     Predicate like(@Nullable Object value);
 
+    /**
+     *Creates a predicate which checks lower than or equals case insensitive.
```

Comment:
Copy&paste error in javadoc.
Please fix accordingly as also the above copy&paste error.

Please also for the `like()` method.